js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -628,6 +628,10 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 			});
 
 			this.pollForInfoPeriodically();
+
+			this.subscriptions.push(
```

Comment:
Thanks for the update: looks like this will refresh print friendly if the underlying cohort definition changes (like if you change the cohort def in the route). Good catch.

I don't think I see this subscription being cleaned up?  Am I missing it?

Reply:
Subscriptions are disposed inside the base `Component`. js/components/Component.js
```
dispose() {
        this.subscriptions.forEach(sub => sub.dispose());
    }
```

Reply:
Thank you thank you...I missed the prevous line of the push! haha, man, I'm getting old :)